lib/db/mysql/index.js

```diff
@@ -153,7 +153,7 @@ MysqlStore.prototype = {
   addAvatar: function addAvatar(id, uid, url, provider, selected) {
     id = buf(id);
     uid = buf(uid);
-    var store = this;
+    var store = this; // eslint-disable-line consistent-this
```

Comment:
@jrgm: as you wish. I moved it from the base, shared "fxa" ESLint config to "fxa/client" (since we mainly enforce it in fxa-content-server). But that's easier than trying to work around it in these other repos. I just published **eslint-config-fxa@1.5.0** to npm and can submit a new PR here to bump the package and remove this noise.